Cargo.toml

```
@@ -17,10 +17,10 @@ keywords = ["gui", "imgui", "immediate", "portable", "gamedev"]
 include = ["../LICENSE-APACHE", "../LICENSE-MIT", "**/*.rs", "Cargo.toml"]
 
 [dependencies]
-bytemuck = "1.7"
-egui = { version = "0.18.1", features = ["bytemuck"] }
-miniquad = { version = "=0.3.0-alpha.46" }
-quad-url = "0.1.0"
+bytemuck = "1.9"
+egui = { version = "0.18", features = ["bytemuck"] }
```

```suggestion
egui = { version = "0.18.1", features = ["bytemuck"] }
```

`egui-miniquad` requires `0.18.1`: https://github.com/emilk/egui/blob/master/CHANGELOG.md#0181---2022-05-01

---

doesn't `0.18.1` get matched with `0.18` too?

---

`0.18` gets matched to either `0.18.0` or `0.18.1`, and only one of those work.